snomed-database-loader/test-load-immunisation-codes.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+set -e;
+
+dbName=patient_switching
+snomedCtSchema=snomedct
+
+rootImmunisationCodes=('787859002' '127785005' '304250009' '90351000119108' '713404003' '127785005')
+childImmunisationCodes=('787859002')
```

Comment:
This immunisation code looks very similar to the first code above, so that's confusing.

Reply:
this was just a test to make sure that jenkins was working as I thought it would. it's been updated to actual values now.